scheduler/managers.py

```diff
@@ -18,9 +18,8 @@ def on_shiftdate(self, shiftdate):
             shiftdate. That means shifts that intersect with the day of
             shiftdate.
         """
-        next_day = datetime.combine(shiftdate + timedelta(days=1), time.min)
         return self.filter(ending_time__gte=shiftdate,
-                           starting_time__lt=next_day)
+                           starting_time__lt=shiftdate + timedelta(days=1))
```

Comment:
Old code set's time of day to zero (`combine` + `time.min`), to filter shifts starting "before midnight".
New code seems to filter for anything starting less than 24 hours after reference.
`shiftdate` currently is assembled only from "year", "month" and "day", but there's no guaranty, no-one will be using "on_shiftdate" with some `datetime` having "time != 0".
IMHO we should explicitly make sure, time is zero.

Reply:
Good point!